### PR TITLE
add a "delegated attestation" use case

### PR DESCRIPTION
**Pull request by thomas-fossati**
Signed-off-by: Thomas Fossati <thomas.fossati@arm.com>

### CODE REVIEW

File: draft-ietf-rats-uccs.md [hunk 1]
```diff
@@ -262,6 +262,11 @@ endorsement of the Secure Channel.  (Note that EAT might add a nested UCCS
 Claim, and this statement does not apply to UCCS nested into UCCS, only to
 fully formed CWTs)
 
+### Delegated Attestation
+
+Another use case is that of an attesting environment that has no signing keys (for example, to keep the implementation complexity to a minimum) and has a secure channel - typically, a local IPC - with a "parent" attesting environment.
+The "child" attesting environment creates a UCCS with the required claims-set and sends it through the secure channel to the parent attester which, in turn, computes a cryptographic hash and protects it using its signing key, for example using a Detached EAT Bundle ({{Section 5 of -eat}}).
```
Comment (henkbirkholz) [inline draft-ietf-rats-uccs.md]:
```suggestion
Another use case is that of a sub-Attester that has no signing keys (for example, to keep the implementation complexity to a minimum) and has a secure channel, such as a local IPC, to interact with a lead Attester (see Composite Device {{Section 3.3 of -rats}}).
The sub-Attester produces a UCCS with the required claims-set and sends the unprotected claims-set through the secure channel to the lead Attester which, in turn, computes a cryptographic hash of the claims-set and protects that hash using its signing key for Evidence, for example, using a Detached EAT Bundle ({{Section 5 of -eat}}).
```

**Review by henkbirkholz — APPROVED** [review]
due to my suggestions, I approve

File: draft-ietf-rats-uccs.md [hunk 2]
```diff
@@ -262,6 +266,12 @@ endorsement of the Secure Channel.  (Note that EAT might add a nested UCCS
 Claim, and this statement does not apply to UCCS nested into UCCS, only to
 fully formed CWTs)
 
+### Delegated Attestation
+
+Another use case is that of a sub-Attester that has no signing keys (for example, to keep the implementation complexity to a minimum) and has a secure channel, such as a local IPC, to interact with a lead Attester (see Composite Device, {{Section 3.3 of -rats}}).
```
Comment (nedmsmith) [inline draft-ietf-rats-uccs.md:271]: SPDM connected components over a system bus such as MCTP is another concrete example

Comment (thomas-fossati) [reply draft-ietf-rats-uccs.md:271]: Arm CCA attestation in "direct" mode another.

**Pull request merged**